lib/engine/game/g_1849/game.rb

```diff
@@ -829,15 +829,15 @@ def par_prices
         # code below is for variant rules
 
         def acquiring_station_tokens?
-          @acquiring_station_tokens ||= @optional_rules&.include?(:acquiring_station_tokens)
+          @optional_rules&.include?(:electric_dreams) || @optional_rules&.include?(:acquiring_station_tokens)
```

Comment:
I don't think this is the right approach. There is a way to select groups of optional rules. See 1825 for inspiration.

Reply:
I feel as though the way 1825 does it is done out of necessity because of the complexity of all the possible combinations. This also works, and is much simpler in my opinion.

Reply:
alright, I modified things after a discussion with crericha over Slack